network-mapper/templates/mapper-clusterrole.yaml

```diff
@@ -52,11 +52,19 @@ rules:
       - ''
     resources:
       - 'configmaps'
-    resourceNames:
-      - {{ template "otterize.mapper.componentConfigmap" . }}
```

Comment:
?

Reply:
To distribute the CA certificate to requesting pods, I create a configMap in each namespace that has pods requesting visibility. `resourceNames` can't be used when creating objects, since the name of the object is not yet known, so I removed this constraint.